diesel/src/sql_types/mod.rs

```diff
@@ -403,16 +403,6 @@ pub trait HasSqlType<ST>: TypeMetadata {
     /// of this method should not do dynamic lookup unless absolutely necessary
     fn metadata(lookup: &Self::MetadataLookup) -> Self::TypeMetadata;
 
-    #[doc(hidden)]
-    #[cfg(feature = "with-deprecated")]
-    #[deprecated(
-        since = "1.4.0",
-        note = "This method is no longer used, and has been deprecated without replacement"
```

Comment:
Something like this is currently required for the oracle backend 😟

Reply:
We should discuss this more and see if there is a better option for this

Reply:
I've opend #1985 for this.